docs/deployment/linux.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Deployment on a Linux Server
+---
+
+This guide will walk you through the installation of Velocity. By the end of the guide, you will have a running instance of Velocity, ready to be configured for your network.
+
+Basic Linux knowledge is assumed.
+
+# Tools
+
+This is a glossary of all the tools and commands we will be using throughout the guide.
+
+These tools are available on most Linux distributions, and many come pre-installed with them. If you aren't sure if they're installed, run the command. If you see `command not found`, you'll need to install them from your package manager.
+
+## GNU Screen
+
+When connecting to a server through SSH, you are limited to running one process at a time. If the SSH session were to disconnect, or you want to run a process in the background, `screen` allows us to do so. We will use `screen` to run the Velocity proxy even if we disconnect from the server.
```

Comment:
The switching from singular "you" to plural "we" in this sentence should all be "you" as that's what you use throughout the document - adding this at your request.
Love,
Larry